hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -686,210 +784,292 @@ public void setOwner(final Path path, final String owner, final String group) th
     }
 
-    LOG.debug(
-            "removeDefaultAcl filesystem: {} path: {}",
-            client.getFileSystem(),
-            path.toString());
+    try (AbfsPerfInfo perfInfoGet = startTracking("removeDefaultAcl", "getAclStatus")) {
 
-    final AbfsRestOperation op = client.getAclStatus(AbfsHttpConstants.FORWARD_SLASH + getRelativePath(path, true));
-    final String eTag = op.getResult().getResponseHeader(HttpHeaderConfigurations.ETAG);
-    final Map<String, String> aclEntries = AbfsAclHelper.deserializeAclSpec(op.getResult().getResponseHeader(HttpHeaderConfigurations.X_MS_ACL));
-    final Map<String, String> defaultAclEntries = new HashMap<>();
+      LOG.debug(
+              "removeDefaultAcl filesystem: {} path: {}",
+              client.getFileSystem(),
+              path.toString());
 
-    for (Map.Entry<String, String> aclEntry : aclEntries.entrySet()) {
-      if (aclEntry.getKey().startsWith("default:")) {
-        defaultAclEntries.put(aclEntry.getKey(), aclEntry.getValue());
+      final AbfsRestOperation op = client.getAclStatus(AbfsHttpConstants.FORWARD_SLASH + getRelativePath(path, true));
+      perfInfoGet.registerResult(op.getResult());
+      final String eTag = op.getResult().getResponseHeader(HttpHeaderConfigurations.ETAG);
+      final Map<String, String> aclEntries = AbfsAclHelper.deserializeAclSpec(op.getResult().getResponseHeader(HttpHeaderConfigurations.X_MS_ACL));
+      final Map<String, String> defaultAclEntries = new HashMap<>();
+
+      for (Map.Entry<String, String> aclEntry : aclEntries.entrySet()) {
+        if (aclEntry.getKey().startsWith("default:")) {
+          defaultAclEntries.put(aclEntry.getKey(), aclEntry.getValue());
+        }
       }
-    }
 
-    aclEntries.keySet().removeAll(defaultAclEntries.keySet());
+      aclEntries.keySet().removeAll(defaultAclEntries.keySet());
 
-    client.setAcl(AbfsHttpConstants.FORWARD_SLASH + getRelativePath(path, true),
-        AbfsAclHelper.serializeAclSpec(aclEntries), eTag);
+      perfInfoGet.registerSuccess(true).finishTracking();
+
+      try (AbfsPerfInfo perfInfoSet = startTracking("removeDefaultAcl", "setAcl")) {
+        final AbfsRestOperation setAclOp =
+                client.setAcl(AbfsHttpConstants.FORWARD_SLASH + getRelativePath(path, true),
+                AbfsAclHelper.serializeAclSpec(aclEntries), eTag);
+        perfInfoSet.registerResult(setAclOp.getResult())
+                .registerSuccess(true)
+                .registerAggregates(perfInfoGet.getTrackingStart(), GET_SET_AGGREGATE_COUNT);
+      }
+    }
   }
 
   public void removeAcl(final Path path) throws AzureBlobFileSystemException {
     if (!getIsNamespaceEnabled()) {
       throw new UnsupportedOperationException(
-          "This operation is only valid for storage accounts with the hierarchical namespace enabled.");
+              "This operation is only valid for storage accounts with the hierarchical namespace enabled.");
```

Comment:
In general all the new lines changed here seem to be double.
Please use the hadoop format style.
https://github.com/cloudera/blog-eclipse/blob/master/hadoop-format.xml

Reply:
Undid this change.